### PR TITLE
master <- rjmurillo/WebView_Controls_WIP

### PR DESCRIPTION
**Pull request by rjmurillo**
**_🌵 🌵 DO NOT MERGE 🌵 🌵_** 

This PR is temporary and meant to capture the latest changes going into `master`.

- For changes going into `master`, see #4 (`Microsoft/UWPCommunityToolkit/master` ⬅️ `rjmurillo/UWPCommunityToolkit/rjmurillo/WebView_Controls`)
- For the delta between `rjmurillo/WebView_Controls` and `rjmurillo/WebView_Controls_WIP` see #3

### CODE REVIEW

File: Microsoft.Toolkit.Win32.UI.Controls/IWebView.cs [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+
+using Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT;
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.Toolkit.Win32.UI.Controls
+{
+    /// <inheritdoc />
```
Comment (rjmurillo) [inline Microsoft.Toolkit.Win32.UI.Controls/IWebView.cs]: @mcleanbyron Most of the documentation for our APIs is on this interface

File: Microsoft.Toolkit.Win32.UI.Controls/Interop/WinRT/WebViewControlAcceleratorKeyPressedEventArgs.cs [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Security;
+
+namespace Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT
```
Comment (rjmurillo) [inline Microsoft.Toolkit.Win32.UI.Controls/Interop/WinRT/WebViewControlAcceleratorKeyPressedEventArgs.cs]: @mcleanbyron All the types I took from WinRT to get our APIs to work without referencing the Windows.winmd file is in this namespace and folder. Documentation should be that of the type mentioned in `seealso` comment. Most places include a remark as to why there is a type that is duplicated with the same name from WinRT (though not this one, I'll clean that up). As I mentioned yesterday, open to renaming these so that there is a clear distinction between the types used by WebBrowser and WebViewControl, as we have plans for an adapter to bridge those two later.

File: docs/controls/WebView.md [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+```
+
+Scripts in the web view content can use **window.external.notify** with a string parameter to send information back to your app. To receive these messages, handle the [ScriptNotify](https://docs.microsoft.com/uwp/api/windows.web.ui.interop.webviewcontrol.scriptnotify) event.
+
+## Options for web content hosting
+
+You can use the [Settings](https://docs.microsoft.com/uwp/api/windows.web.ui.interop.webviewcontrol.settings) property (of type [WebViewControlSettings](https://docs.microsoft.com/uwp/api/windows.web.ui.interop.webviewcontrolsettings) to control whether JavaScript and IndexedDB are enabled. For example, if you use a web view to display strictly static content, you might want to disable JavaScript for best performance.
+
+ 
+
```
Comment (nmetulev) [inline docs/controls/WebView.md]: Add Requirements section from [template](https://github.com/Microsoft/UWPCommunityToolkit/blob/master/docs/.template.md)

File: docs/toc.md [hunk 4]
```diff
@@ -60,6 +60,7 @@
 ## [StaggeredPanel](controls/StaggeredPanel.md)
 ## [TextToolbar](controls/TextToolbar.md)
 ## [TileControl](controls/TileControl.md)
+## [WebView](controls/WebView.md)
```
Comment (nmetulev) [inline docs/toc.md]: Should we add this to a new category called Win32 controls?

File: build/build.ps1 [hunk 5]
```diff
@@ -181,9 +184,49 @@ if (!(Test-Path $CAKE_EXE)) {
     Throw "Could not find Cake.exe at $CAKE_EXE"
 }
 
+# Make sure modules folder exists
```
Comment (nmetulev) [inline build/build.ps1]: Why these changes?

Comment (rjmurillo) [reply build/build.ps1]: @nmetulev See dcd5aff451fdbd1d7e7e77d785c55071145c4f23. Later replaced with 174ee69cc1abb8ed11658c18580464bb7c070745 but maintained change. Can revert this portion since I'm using the directive introduced with the latter.

File: docs/toc.md (hunk 4 above)
Comment (nmetulev) [reply docs/toc.md]: Or simply specify in the name that this is a win32 control ex: `WebvView (win32)`
